mycli/completion_refresher.py

```diff
@@ -1,21 +1,26 @@
-# type: ignore
+from __future__ import annotations
 
-from collections import OrderedDict
 import threading
+from typing import Callable
 
 from mycli.packages.special.main import COMMANDS
 from mycli.sqlcompleter import SQLCompleter
 from mycli.sqlexecute import ServerSpecies, SQLExecute
 
 
 class CompletionRefresher:
-    refreshers = OrderedDict()
+    refreshers: dict = {}
```

Comment:
Changing from OrderedDict to Dict didn't change behavior? 

I'm trying to remember why this was an OrderedDict to begin with.

Reply:
It shouldn't change behavior because `dict` is ordered after Python 3.7.  There are some small differences with OrderedDict per https://docs.python.org/3/library/collections.html#collections.OrderedDict , but one difference is that `dict` is supposed to be more performant for the common case.